### ISSUE TITLE
Use listbox role for BE records lists

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2192]
Users should be able to easily navigation in and out of, and between, list items using a keyboard. Using `role="listbox"` should resolve this issue.

**Comment by dananjohnson** [comment]
I believe this incorrect, actually. As I understand it, a listbox provides a user with a list of static options, similar to the `select` element. But these are lists of links to records (and include searchability, filters, and pagination).

Maybe there are other ways we can improve accessibility here, such as adding a heading or `aria-label` attribute to each list. Some guidance would be helpful.

**Comment by zdavis** [comment]
@Santoz2 @avaaec — we're a little stuck on this issue. This comes from the first round of feedback from UW, I think. The role we're being asked to add here doesn't seem right to us, so we just want to make sure that we're on the right track.

@dananjohnson Could you help out our friends at UW by including a URL (on edge) for a representative instance of this list, as well as a screenshot of the component.

**Comment by dananjohnson** [comment]
Sorry to overlook this earlier. @Santoz2 @avaaec — see [here](https://edge.manifoldapp.org/backend/projects/0774268b-37bb-4e8d-acb5-ef16a3e48e8e/resources) for an example of such a list. It looks like the following:

<img width="970" alt="Screen Shot 2019-07-29 at 3 49 46 PM" src="https://user-images.githubusercontent.com/12454203/62088063-d4c2b480-b218-11e9-8a5d-893e47c86273.png">

**Comment by avaaec** [comment]
Hi all, apologies on the delay with this response.
I believe that the list of results is implemented correctly; however, this issue may more specifically apply to the filtering options involved with this search. Currently these controls announce as combo boxes to screen readers. They should be list boxes.

An example of a proper implementation of a listbox can be seen here: 
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

## Steps To Reproduce
1. Go to above link
2. navigate to search box
3. Select options button
4. navigate to one of the options

**Comment by dananjohnson** [comment]
Thanks for following up, @avaaec! The search filters are implemented using the native `<select>` element, so I'm unsure why they would be announced as combo boxes. Can you provide more specific feedback about what's wrong with our markup? Thank you!

**Comment by avaaec** [comment]
@dananjohnson my bad! I did some research I should have done before updating this issue and realized that the problem actually lies in some assistive tech not being fully compatible with html5 select yet. You have followed the best practices for accessible select elements, this issue can be closed I think.

**Comment by dananjohnson** [comment]
No problem @avaaec! Thanks for following up. We'll go ahead and close this issue.
